src/amp/lib/experiment.ts

```diff
@@ -0,0 +1,137 @@
+interface ExperimentModel<VariantData> {
```

Comment:
sometimes more natural language for names can make them easier to read when you first see the code (if it doesn't remove too much context). e.g.
```suggestion
interface Experiment<Variant> {
```

Reply:
This is a great spot and suggestion. Thanks